resources/views/components/head-analytics.blade.php

```diff
@@ -53,6 +54,19 @@
             break;
         }
     }
+
+    // Extract the `t` parameter from gameList.php to track what kind of list users are opening.
+    if (str_ends_with($url, 'gameList.php') && isset($queryParams['t'])) {
+        $tParam = $queryParams['t'];
+        if ($tParam === 'play' || $tParam === 'develop') {
+            $props['type'] = $queryParams['t'];
+        }
+    }
+
+    // Track what topic ID users are viewing at viewtopic.php.
+    if (strpos($url, 'viewtopic') !== false && isset($queryParams['t'])) {
+        $props['topicId'] = $queryParams['t'];
```

Comment:
I don't think it's worth capturing the current page for multi-page topics, but we need to be aware that "hot" topics might just be because users are scanning through pages of posts.

Reply:
This is true - though likely still a valuable datapoint. We may find there are specific topics that people are likely to spend time scanning through (or not). Either way, a good thing to keep in mind as we parse this data.